tests/EdFi.Ods.Api.IntegrationTests/EducationOrganizationTestDataBuilder.cs

```diff
@@ -168,16 +176,19 @@ public EducationOrganizationTestDataBuilder AddStudentSchoolAssociation(int scho
         {
            _sql.AppendLine(
                 $@"INSERT INTO edfi.StudentSchoolAssociation (SchoolId,StudentUSI,EntryDate,EntryGradeLevelDescriptorId)
-                VALUES (" + schoolId + "," + studentUSI + ",'" + DateTime.UtcNow.Date + "'," + gradeLevelDescriptorId + ");");
-
+                VALUES (
+                    {schoolId},{studentUSI},
+                    '{DateTime.UtcNow.Date}',{gradeLevelDescriptorId});"
+            );
```

Comment:
should each of these column names be on a new line to be consistent with the other sql in AddStudent()?

Reply:
done